### PR TITLE
Update more page links

### PR DESCRIPTION
**Pull request by skoech**
Signed-off-by: Sharon Koech <sharonkoech5147@gmail.com>

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ module.exports = {
     footer: {
       logo: {
         alt: 'Tremor Logo',
-        src: 'static/favicon.ico',
+        src: '/img/favicon.ico',
```
Comment (Licenser) [inline docusaurus.config.js:67]: this should probably just be `/favicon.ico`

**Review by Licenser — CHANGES_REQUESTED** [review]
Looks great, only the favicon thingy. It should either be on the top level (like it was) or we need to configure it, not sure how to do that in docusaurus but I'm sure it's possible :)

File: docusaurus.config.js (hunk 1 above)
Comment (skoech) [reply docusaurus.config.js:67]: Oh, I didn't know that. Thanks.

**Review by Licenser — APPROVED** [review]
🚀 awesome :)

**Pull request merged**